extensions/crossmodel-lang/src/language-server/relationship.langium

```diff
@@ -4,20 +4,24 @@ import 'entity'
 // Relationship defintion
 Relationship:
     'relationship' ':' 
-    (
         INDENT
-            RelationshipFields*
+        (
+            'id' ':' id=ID |
+            'name' ':'  name=STRING |
+            'description' ':' description=STRING |
+            'parent' ':' parent=[Entity:IDReference] |
+            'child' ':' child=[Entity:IDReference] |
+            'type' ':' type=STRING |
+            ('attributes' ':'
+                INDENT 
+                    attributes+=RelationshipAttribute+
+                DEDENT)
+        )*
         DEDENT
-    )*
 ;
-
-RelationshipFields infers Relationship:
-    (
-        'id' ':' id=ID |
-        'name' ':'  name=STRING |
-        'description' ':' description=STRING |
-        'parent' ':' parent=[Entity:IDReference] |
-        'child' ':' child=[Entity:IDReference] |
-        'type' ':' type=STRING
-    )
-;
\ No newline at end of file
+ 
+RelationshipAttribute:
+    '-' 'parent' ':' parent=[Attribute:IDReference]
+        'child' ':' child=[Attribute:IDReference]
+        ('description' ':' description=STRING)?
```

Comment:
This description is better to remove. This should never be used.

Reply:
I removed it, thank you!